ansible/roles/compute_init/files/compute-init.yml

```diff
@@ -132,6 +142,93 @@
         mode: 0644
       when: enable_etc_hosts
 
+    - name: Configure sssd
```

Comment:
Looking at the role, it looks like it'd just be enough to run the configure.yml task, overriding `sssd_conf_src`, if we import the role. See comment for sshd below. However this does look ok